xapian-bindings/configure.ac

```diff
@@ -94,6 +94,24 @@ XO_LIB_XAPIAN([], [],
 			  [^\([0-9]*\.[0-9]*[13579]\)\..*$], [-\1]))
 XO_REQUIRE([1.4.0])
 
+AC_ARG_ENABLE([letor],
+  [AS_HELP_STRING([--enable-letor], [])],
```

Comment:
This has an empty help string, so `configure --help` says nothing about what this option does, unlike all the options around it:

```
  --disable-dependency-tracking
                          speeds up one-time build
  --disable-libtool-lock  avoid locking (might break parallel builds)
  --enable-letor
  --disable-visibility    disable use of GCC visibility
  --enable-maintainer-mode
                          enable make rules and dependencies not useful (and
                          sometimes confusing) to the casual installer
  --enable-documentation  enable make rules to rebuild shipped documentation
                          [default=maintainer-mode]
```
```

Reply:
fixed in a fixup commit.